robottelo/helpers.py

```diff
@@ -1080,8 +1080,10 @@ def _handle_version(key, value):  # pragma: no cover
     return value
 
 
-def download_gce_cert(gce_cert_url, gce_cert_path):
-    ssh.command('curl {0} -o {1}'.format(gce_cert_url, gce_cert_path))
-    if ssh.command('[ -f {} ]'.format(gce_cert_path)).return_code != 0:
+def download_gce_cert():
+    ssh.command('curl {0} -o {1}'.format(settings.gce.cert_url, settings.gce.cert_path))
+    if not ssh.command('[ -f {} ]'.format(settings.gce.cert_path)).return_code == 0:
```

Comment:
In this line, previous way was more natural. It was "if command return is not equal zero" instead of "if not command return equals zero".